samples/join-a-team-using-qr-code/csharp/README.md

```diff
@@ -0,0 +1,110 @@
+---
+page_type: sample
+description: This sample demos a feature where user can join a team using QR code containing the team's id.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "24-12-2021 23:35:25"
+---
+
+# Join a team using QR code sample
+
+This sample demos a feature where user can join a team using QR code containing the team's id.
+
+User can generate a new QR code (contains team id information) and then scan the QR code to join the team.
+
+`Currently, Microsoft Teams support for QR or barcode scanner capability is only supported for mobile clients`
+
+- Type a message to get a card to generate the QR code.
+
+ ![Card](JoinTeamByQR/Images/CardWithButtons.png)
+
+- Select the team from dropdown list for which you want to generate the QR code and then click on      'Generate QR' button.
+
+ ![QR Code](JoinTeamByQR/Images/QRCode.png)
+
+- Scan the generated QR code to join the team.
+
+ ![Join Team](JoinTeamByQR/Images/TeamQR.png)
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+- Publicly addressable https url or tunnel such as [ngrok](https://ngrok.com/) or [Tunnel Relay](https://github.com/OfficeDev/microsoft-teams-tunnelrelay) 
+
+## Setup
+
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+2. In a terminal, navigate to `samples/join-a-team-using-qr-code/csharp`
```

Comment:
Project folder name is still wrong "join-a-team-using-qr-code"
Please rename it as "bot-join-a-team-using-qr-code"